### PR TITLE
4 7 transscript

### LINKED ISSUE #52: Improve episode transcription Season 4 Episode 7
## Issue Context

Our podcast's transcriptions are automatically generated, so there would be typos or missing words. 

We want to improve every episode's transcription so they can be accessible to everyone. Currently, we are doing this manually.

## Steps To Update

- Open the file of the podcast's episode. 
  e.g.: `1_0.srt` means season 1 episode 0
-  Listen to the podcast's episode and improve the transcript based on your hearing.
-  [Fix the format](https://github.com/Virtual-Coffee/podcast-transcripts#fixing-formatting-issues) by running `yarn srt-check`.
- After running `yarn srt-check`, fix timestamps suggestions format manually, if any.

## Accessibility Resources

- [Transcribing Audio to Text - W3C WAI](https://www.w3.org/WAI/media/av/transcribing/)

If you have questions or need help, please let us know.

### PR DESCRIPTION
**Pull request by ClJarvis**
Improve transcription Season 4 Episode 7
## Description

This PR improves transcript Season 4 Episode 7 with Jessica Wilkins.  Growing your tech career through writing.

## Related Issue

Closes #52

### CODE REVIEW

**Comment by adiati98** [comment]
Hey @ClJarvis,
Thanks for your PR!

You added there are two episodes files here.
Can you please remove the Season 3 Episode 9 as it's not related to this PR?

Thank you! :)

**Comment by ClJarvis** [comment]
@adiati98 deleted file.

**Comment by adiati98** [comment]
@ClJarvis I see a conflicting files here. Please resolve it first :)
I think you should `git pull` the branch from your remote to your local, then push it back. 

Try this:

checkout via command line

**Comment by adiati98** [comment]
@ClJarvis it might be easier for you to copy paste the commands.

Click the "Use the command line" link in the conflicts box here. It will open the commands as my previous screenshot.

use the command line

File: episodes/4_7.srt [hunk 1]
```diff
@@ -1,5635 +1,5856 @@
...
 00:09:25,158 --> 00:09:29,957
-happened. I wildly miss,
-understood how COVID works. I
+happened. I wildly misunderstood
+how COVID works. I
 
-191
-00:09:29,957 --> 00:09:32,388
+204
+Bekah: same
```
Comment (adiati98) [inline episodes/4_7.srt:952]: Why there is no timestamp here?

File: episodes/4_7.srt [hunk 2]
```diff
@@ -1,5635 +1,5856 @@
...
+
+205
+00:09:31,005 --> 00:09:32,388
+Dan: everyone did.
+
+206
 totally thought this was just
```
Comment (adiati98) [inline episodes/4_7.srt:959]: There is also no timestamp here and the speaker here is Jessica.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @ClJarvis,

I found conversations without timestamps and without or not correct speakers.

There are also many typos. For example, the first letter after a period should be capitalized, "FreeCodeCamp" should be "freeCodeCamp", etc.

I haven't checked until the end. But please re-check everything and fix anything necessary.

@danieltott will update the transcripts on buzzsprout. So timestamps and speakers are very important here.

Thank you :)

**Comment by ClJarvis** [comment]
> @ClJarvis it might be easier for you to copy paste the commands.
> 
> Click the "Use the command line" link in the conflicts box here. It will open the commands as my previous screenshot.
> 
> use the command line

Where is the conflicts box?

**Comment by adiati98** [comment]
> > @ClJarvis it might be easier for you to copy paste the commands.
> > 
> > Click the "Use the command line" link in the conflicts box here. It will open the commands as my previous screenshot.
> > 
> > use the command line
> 
> Where is the conflicts box?

You should see it when you scroll down. If you don't, you can refer to my other screenshot.

**Comment by ClJarvis** [comment]
> > > @ClJarvis it might be easier for you to copy paste the commands.
> > > Click the "Use the command line" link in the conflicts box here. It will open the commands as my previous screenshot.
> > > use the command line
> > 
> > 
> > Where is the conflicts box?
> 
> You should see it when you scroll down. If you don't, you can refer to my other screenshot.

Oh it's grayed out and disabled for me.

**Comment by adiati98** [comment]
> > > > @ClJarvis it might be easier for you to copy paste the commands.
> > > > Click the "Use the command line" link in the conflicts box here. It will open the commands as my previous screenshot.
> > > > use the command line
> > > 
> > > 
> > > Where is the conflicts box?
> > 
> > You should see it when you scroll down. If you don't, you can refer to my other screenshot.
> 
> Oh it's grayed out and disabled for me. 
> 
> 

As I mentioned in the previous post, click the link that I highlighted.

**Comment by ClJarvis** [comment]
Maybe I was unclear. That is grayed out and disabled for me. I do not have write access to it. 

I removed the extra file. Made changes to 4_7 and submitted another PR. 

conflicts

**Comment by adiati98** [comment]
Thanks for the clarification, @ClJarvis!
I will close this PR as it's moved to #111. :)

**Pull request closed without merging**